kyaml/yaml/fns.go

```diff
@@ -541,6 +541,9 @@ func (l PathGetter) getFilter(part, nextPart string, fieldPath *[]string) (Filte
 	case part == "-":
 		// part is a hyphen
 		return GetElementByIndex(-1), nil
+	case part == "*":
+		// part is a hyphen
+		return nil, fmt.Errorf("cannot support create option in a multi-value target now")
```

Comment:
change `fmt.Errorf` to `errors.New` to fix the lint error

Reply:
I fix it.